cadquery/assembly.py

```diff
@@ -466,8 +466,10 @@ def save(
         :param exportType: export format (default: None, results in format being inferred form the path)
         :param tolerance: the deflection tolerance, in model units. Only used for GLTF, VRML. Default 0.1.
         :param angularTolerance: the angular tolerance, in radians. Only used for GLTF, VRML. Default 0.1.
-        :param \**kwargs: Additional keyword arguments.  Only used for STEP.
+        :param \**kwargs: Additional keyword arguments.  Only used for STEP, GLTF and STL.
```

Comment:
I think it should be changed here in the docstring too for param kwargs: GLTF -> glTF
Also for tolerance, angularTolerance.